src/main/resources/config.properties

```diff
@@ -0,0 +1,2 @@
+test.username=user	
```

Comment:
This file shouldn't be committed if it has passwords in it (even if this is a dummy password, someone might change it on their local machine and accidentally commit it).